chain/beacon/beacon.go

```diff
@@ -79,7 +80,12 @@ func ValidateBlockValues(bSchedule Schedule, nv network.Version, h *types.BlockH
 		return xerrors.Errorf("expected to have beacon entries in this block, but didn't find any")
 	}
 
-	// Verify that the last beacon entry's round corresponds to the round we expect
+	if nv < network.Version22 && prevEntry.Round == 0 {
```

Comment:
Ditto, should we decouple drand network version and chain network version here?

Reply:
Similar to above, I don't think so, no. From nv22 onwards we'll be using "unchained" randomness, and so can simply verify round 0 too.